api-admin/src/main/java/bio/terra/pearl/api/admin/controller/dataimport/DataImportController.java

```diff
@@ -0,0 +1,52 @@
+package bio.terra.pearl.api.admin.controller.dataimport;
+
+import bio.terra.pearl.api.admin.api.DataImportApi;
+import bio.terra.pearl.api.admin.service.AuthUtilService;
+import bio.terra.pearl.api.admin.service.enrollee.EnrolleeImportExtService;
+import bio.terra.pearl.core.model.admin.AdminUser;
+import jakarta.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.multipart.MultipartFile;
+
+@Controller
+public class DataImportController implements DataImportApi {
+  private AuthUtilService authUtilService;
+  private HttpServletRequest request;
+  private EnrolleeImportExtService enrolleeImportExtService;
+
+  public DataImportController(
+      AuthUtilService authUtilService,
+      HttpServletRequest request,
+      EnrolleeImportExtService enrolleeImportExtService) {
+    this.authUtilService = authUtilService;
+    this.request = request;
+    this.enrolleeImportExtService = enrolleeImportExtService;
+  }
+
+  @Override
+  public ResponseEntity<Object> delete(
+      String portalShortcode, String studyShortcode, String envName, Integer importId) {
+    return DataImportApi.super.delete(portalShortcode, studyShortcode, envName, importId);
+  }
+
+  @Override
+  public ResponseEntity<Object> get(String portalShortcode, String studyShortcode, String envName) {
+    AdminUser operator = authUtilService.requireAdminUser(request);
+    return ResponseEntity.ok(enrolleeImportExtService.list(portalShortcode, operator));
+  }
+
+  @Override
+  public ResponseEntity<Object> callImport(
```

Comment:
```suggestion
  public ResponseEntity<Object> import(
```

Reply:
the 'call' is redundant in the method name

Reply:
swagger generated API code had callImport (even though Api spec has just import), hence controller need 'call'

Reply:
ooh, I wonder if it's because `import` is a reserved word in Java -- maybe call this `importData` to match the ext service method name?

Reply:
changed to dataImport